modules/ROOT/pages/whats_new_rpcn.adoc

```diff
@@ -0,0 +1,139 @@
+* The `job_project` field to the xref:components:outputs/gcp_bigquery.adoc[`gcp_bigquery` output].
+
+== Version 4.34.0
+
+Redpanda Connect now includes beta versions of two new AI processors for Cloud and Self-Managed platforms:
+
+* xref:components:processors/aws_bedrock_chat.adoc[`aws_bedrock_chat`]
+* xref:components:processors/gcp_vertex_ai_chat.adoc[`gcp_vertex_ai_chat`]
+
+== Version 4.33.0
+
+Redpanda Connect now includes beta versions of the following components for Self-Managed and Cloud platforms:
+
+* xref:components:inputs/schema_registry.adoc[`schema_registry` input] and xref:components:outputs/schema_registry.adoc[output]
+* xref:components:outputs/qdrant.adoc[`qdrant` output]
+
+You can also use the new Bloblang xref:guides:bloblang/methods.adoc[`vector`] method to create vector embeddings for SQL databases. 
+
+=== Component updates
+
+This release adds:
+
+* The `credentials_json` field to all GCP components.
+* The `content_md5` field to the xref:components:outputs/aws_s3.adoc[`aws_s3` output].
+* The `send_ack` field to the xref:components:inputs/nats.adoc[`nats` input].
+
+== Version 4.32.1
+
+Redpanda Connect has a new batch of beta AI processors for Self-Managed and Cloud platforms.
+
+=== For Self-Managed and Cloud platforms
+
+* xref:components:processors/openai_chat_completion.adoc[`openai_chat_completion`]
+* xref:components:processors/openai_embeddings.adoc[`openai_embeddings`]
+* xref:components:processors/openai_image_generation.adoc[`openai_image_generation`]
+* xref:components:processors/openai_speech.adoc[`openai_speech`]
+* xref:components:processors/openai_transcription.adoc[`openai_transcription`]
+* xref:components:processors/openai_translation.adoc[`openai_translation`]
+
+=== For Self-Managed only
+
+* xref:components:processors/ollama_chat.adoc[`ollama_chat`]
+* xref:components:processors/ollama_embeddings.adoc[`ollama_embeddings`]
+
+
+== Version 4.31.0
+
+Redpanda Connect now includes beta versions of two new components:
+
+* xref:components:outputs/pinecone.adoc[`pinecone` output] - Redpanda Connect's first AI component 
+* xref:components:processors/redpanda_data_transform.adoc[`redpanda_data_transform` processor]
+
+
+=== Component updates
+
+This release adds:
+
+* Custom TLS fields for the xref:components:inputs/splunk.adoc[`splunk` input] and xref:components:outputs/splunk_hec.adoc[`splunk_hec` output].
+* The `timestamp` field to xref:components:outputs/kafka.adoc[`kafka`] and xref:components:outputs/kafka_franz.adoc[`kafka_franz`] outputs.
+* The `status_topic` field to the xref:components:redpanda/about.adoc[`redpanda` component].
+
+== Version 4.30.0 to 4.30.1
+
+Redpanda Connect now includes a beta version of the xref:components:inputs/splunk.adoc[`splunk` input].
+
+== Version 4.29.0
+
+Redpanda Connect now separates packages into enterprise and free bundles.
```

Comment:
Seems like we should explain what the difference between enterprise and free is here.

Reply:
Added more details and linked to supported connectors page for more information.